website/src/pages/docs/typescript.js

```diff
@@ -24,18 +24,18 @@ const Installation = () => {
       </p>
 
       <p className="Docs__paragraph">
-         You can use <code className="custom-code">Day</code> and <code className="custom-code">DayRange</code> types that has been exported by this package to define generic type variable.
+        You can use <code className="custom-code">Day</code> and <code className="custom-code">DayRange</code> types that has been exported by this package to define generic type variable.
       </p>
 
       <Code language="javascript">{`
 import React from 'react'
 import DatePicker, { DayValue, DayRange, Day } from 'react-modern-calendar-datepicker'
 
 function App() {
-  const [day, setDay] = React.useState<DayValue>(null);
+  const [day, setDay] = React.useState<DayValue>(undefined);
   const [dayRange, setDayRange] = React.useState<DayRange>({
-    from: null,
-    to: null
+    from: undefined,
+    to: undefined
```

Comment:
This example brakes in runtime, looks like ranged date picker doesn't work with undefined values! (You can see a living example in the [CodeSandBox](https://codesandbox.io/s/react-modern-calendar-datepicker146-m5srq?file=/src/index.js))
@Kiarash-Z should consider it as a bug or a feature? :/

Reply:
It should work and it doesn't. I think it's a bug report. Can you open an issue for this, please?

Reply:
sure thing.